### ISSUE TITLE
git submodule update: error on LaZagne

### ISSUE THREAD

**Issue by nil0x42** [n1nj4sec/pupy#237]
### Issue
Hi, after cloning pupy for the first time and taking a look at the wiki, i decided to start using the tool,
so i ran git submoules initialization and got an error.
Is it expected ? Will some functionnality be absent ?

Thanks in advance.

### Commands
`$ git submodule init`
```
Submodule 'client/sources-linux/linux-inject' (https://github.com/gaffe23/linux-inject.git) registered for path 'client/sources-linux/linux-inject'
Submodule 'pupy/external/LaZagne' (https://github.com/AlessandroZ/LaZagne) registered for path 'pupy/external/LaZagne'
Submodule 'pupy/external/impacket' (https://github.com/CoreSecurity/impacket) registered for path 'pupy/external/impacket'
Submodule 'pupy/external/pywerview' (https://github.com/the-useless-one/pywerview.git) registered for path 'pupy/external/pywerview'
Submodule 'pupy/packages/src/scapy' (https://github.com/secdev/scapy) registered for path 'pupy/packages/src/scapy'
Submodule 'pupy/payload_templates' (git@github.com:n1nj4sec/pupy-binaries.git) registered for path 'pupy/payload_templates'
```

`$ git submodule update`
```
Cloning into '/home/user/pupy/client/sources-linux/linux-inject'...
Cloning into '/home/user/pupy/pupy/external/LaZagne'...
Cloning into '/home/user/pupy/pupy/external/impacket'...
Cloning into '/home/user/pupy/pupy/external/pywerview'...
Cloning into '/home/user/pupy/pupy/packages/src/scapy'...
Cloning into '/home/user/pupy/pupy/payload_templates'...
Submodule path 'client/sources-linux/linux-inject': checked out 'ce6d7e4b1d6c06b2fb36548e23d62685a1f91209'
error: no such remote ref adf982740b40628e13b11aeb9b2844ca3b72ab21
Fetched in submodule path 'pupy/external/LaZagne', but it did not contain adf982740b40628e13b11aeb9b2844ca3b72ab21. Direct fetching of that commit failed.
```

**Comment by domecort** [comment]
fix Lazagne head #191 , even though i don't know how to change it. anyone? can help pls?

**Comment by n1nj4sec** [comment]
man git
